### PR TITLE
Enforce coding style on Station

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

File: src/OpenLoco/Map/Tile.h [hunk 1]
```diff
@@ -194,9 +195,9 @@ namespace OpenLoco::Map
 
     public:
         uint8_t objectId() const { return _5 & 0x1F; }
-        uint8_t stationType() const { return _5 >> 5; }
+        StationType stationType() const;
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Map/Tile.h:198]: Why this change? I think the original idea was that declaring these accessor members here would enable more inlining or local optimisations.

Comment (duncanspumpkin) [reply src/OpenLoco/Map/Tile.h:198]: To save having to include station in the header. These days link time optimisations usually catch anything that could be inlined like this

Comment (AaronVanGeffen) [reply src/OpenLoco/Map/Tile.h:198]: Ah, I see. Probably worth it then.

**Review by AaronVanGeffen — APPROVED** [review]
